tools/replaceColumn/replaceColumn.xml

```diff
@@ -14,35 +14,46 @@ replace_file = '$replace_information'
 original_file = '$original_file'
 column = int("$column_replace") - 1
 ignore_start_lines = int("$skip_lines")
-delimiter_local = "\t" if str("$delimiter") == "" else str("$delimiter")
+delimiter_local = "\t" if len(str("$delimiter")) != 1 else str("$delimiter")
```

Comment:
I'd replace `len(str("$delimiter")) != 1` with `str("$delimiter") == 'tab'` .

Reply:
If I do that, when `$delimiter` is "", `delimiter_local` will get "" and the join will be on an empty string when it "should" be `\t`.  There are currently 2 values that result in `\t` being selected as the join character (i.e. `delimiter_local`): empty string and `tab`.  If I understand the python if/else correctly (and I could be wrong because I've never seen this style before), the `if` needs to account for both cases - unless we remove whitespace as an option.

Reply:
You're right. On the other hand, I'd also vote to remove the "any whitespace" option.

Reply:
Alright, well we're unanimous about removing the whitespace option it seems. I'll go ahead and do that.

Reply:
Done.  Note, I had to change one of the test files which relied on the space-split/tab-join behavior.